crates/astria-composer/src/executor/bundle_factory/mod.rs

```diff
@@ -74,6 +76,12 @@ impl SizedBundle {
         }
     }
 
+    #[allow(clippy::panic)] // method is expected to never panic as an invariant of the type
```

Comment:
because the actions are only ever of the `SequenceAction` type?

Reply:
made more descriptive